torch/nn/init.py

```diff
@@ -422,10 +432,11 @@ def kaiming_uniform_(
 def kaiming_normal_(
     tensor: Tensor, a: float = 0, mode: str = 'fan_in', nonlinearity: str = 'leaky_relu'
 ):
-    r"""Fills the input `Tensor` with values according to the method
-    described in `Delving deep into rectifiers: Surpassing human-level
-    performance on ImageNet classification` - He, K. et al. (2015), using a
-    normal distribution. The resulting tensor will have values sampled from
+    r"""Fill the input `Tensor` with values using a normal distribution.
```

Comment:
The first docstring line should include "kaiming" or something like that.
As is, the first line is exactly the same as for `xavier_normal_`, which is not good.